### ISSUE TITLE
Add folders = "all-known" or export 'known_folders' (?)

### ISSUE THREAD

**Issue by HenrikBengtsson** [eddelbuettel/ciw#1]
How do I tell `incoming()` to return all known folders? I'm currently doing:

```r
db <- ciw::incoming(folder = ciw:::known_folders, check = FALSE)
```

Would it make sense to export `known_folders`, or maybe add a `folders = "all-known"` value?

**Comment by eddelbuettel** [comment]
Feedback on `ciw`  :heart_eyes:   I really thought I was the only one :)

I am not sure all known folders is a well-posed question. I would expect this to change over time, and I (currently) do not read it.  (One could, I suppose.)

I call this from the [wrapper `ciw.r`](https://github.com/eddelbuettel/littler/blob/master/inst/examples/ciw.r) I wrote for it. That as two meta-arguments `auto` (which is the one I prefer) and one `mega` (even more).  Both stop short of the folders `BA`, `KH`, `KL`, `UL`, `VW` some of which I expect to possible disppear.   But given what `ciw.r` does, I think the answer to your question is to create a named vector `all_folder` at your end and pass that.

**Comment by eddelbuettel** [comment]
The above works.  

```r
> dim(ciw::incoming(ciw:::known_folders, check=FALSE, ping=FALSE))    
[1] 263   5          
>  
```

So I think we're at the 'Dirk does not think making this a default is sensible.  The key folders that `ciw::incoming()` is after are typically only one directory deep, but some of these others are not (`UL` for example) which renders (non-recursively) looping over them a little moot. 

Do you really think we need "all"  ?

**Comment by eddelbuettel** [comment]
So I guess you would be helped if package `ciw` directly had what the wrapper does via `-m | --mega` ?

> Folder selecting arguments are cumulative. Selecting '-m' or '--mega' is a single selections
of all folders that are not per-user folders (i.e. 'inspect', 'waiting', 'pending', 'pretest',
'recheck', 'archive', 'newbies', 'publish').

Preference for a new function around `incoming` or a new argument to `incoming` ? Maybe `most` ?  I still think that selecting `known_folders[1:8]` makes more sense than all of `known_folder`.   

Thoughts?

**Comment by HenrikBengtsson** [comment]
(On my phone in a car)

My use case is https://www.cranhaven.org/dashboard-live.html. Search for "resubmitted".

Some packages submitted get assigned to specific CRAN members, so want to convey that info too so it's clear where suc packages sit in the incoming queue.

FWIW, https://nx10.github.io/cransubs group those "initials" folder into a "other" folder

**Comment by eddelbuettel** [comment]
Well that seems like a bit of special request for special folders which in an editorial decision made when I wrote this I considered too specialised.  And I don't particularly care what other websites do.   I find the older default dashboard unwieldy (the cut off at 50 is arbitrary) so I found myself visiting the folders (that I now frequent _programmatically_  which is better) in a mechanical way via the browser and refreshing.  Which was suboptimal, hence the new helper.  And Florian's is just ... too slow for my taste.  YMMV.

What I write does what I want it to do, and I am not (yet ?) convinced that what you suggest adds values so I am leaning towards closing this.   If you have a real itch to scratch here, consider maybe a minimal patch demonstrating the usefulness.  But I don't think I will write it -- sorry.
